contrib/test/integration/build/cri-o.yml

```diff
@@ -82,10 +82,9 @@
 
 - name: add quay.io and docker.io as default registries
   copy:
-    dest: /etc/crio/crio.conf.d/01-registries.conf
+    dest: /etc/containers/registries.conf
```

Comment:
I've a vague memory that CRI-O wasn't supposed to use this file?  Bad memory, or have things changed?

Reply:
I only stumbled upon it when combing through failed CI logs, saw this warning message, then git-grepped for it.

https://github.com/cri-o/cri-o/blob/93cdb4600d8a1332fcad08ea89ad6cf3b8f3ec2a/pkg/config/config.go#L631